gulpfile.js

```diff
@@ -33,8 +37,7 @@ gulp.task('images', function () {
       progressive: true,
       svgoPlugins: [{removeViewBox: false}],
       use: [pngquant()]
```

Comment:
can we get rid of of use: [pngquant()]

I found out that is where my gulp is breaking and it compresses images without it.
http://alfanso.com/compressing-your-images-using-gulp-js/